extern/sector-storage/stores/index.go

```diff
@@ -51,15 +51,33 @@ type SectorStorageInfo struct {
 	Primary bool
 }
 
+// SectorIndex is the Miner's sector store Index/Registry that keeps track of all storage directories
+// and sector files in the sector stores of all Workers.
+// The Miner exposes this Index API over JSON RPC to all remote workers. Local workers can of-course call it from within the process.
 type SectorIndex interface { // part of storage-miner api
+	// StorageAttach is called by workers to register a new storage directory or update information about an existing storage directory
+	// with the Index.
 	StorageAttach(context.Context, StorageInfo, fsutil.FsStat) error
+	// StorageInfo is used to get detailed information about a storage directory with the given ID.
 	StorageInfo(context.Context, ID) (StorageInfo, error)
+	// StorageReportHealth is called by workers to report the health of a registered storage directory.
+	// This is the mechanism my which the Index tracks healthy/unhealthy/available/unavailable storage directories across workers.
```

Comment:
Same as above, let's separate function from usages. A problem with this style is that this is a public API, and it may get used in other ways in the future, which would render this documentation obsolete immediately. So the better style here is:

```
StorageReportHealth records the health ... (function)

It is used by ..., at least. (usage known now)
```